src/dreyfus_index.erl

```diff
@@ -40,6 +40,9 @@
     waiting_list=[]
 }).
 
+-define(INDEX_SHUTDOWN_DELAY,
+    config:get("dreyfus", "index_shutdown_delay", 60000)). % 1 minute
```

Comment:
config:get returns a string, so the default needs to be a string, or use get_integer. And move this to a function please.